docs/rest_api.md

```diff
@@ -533,7 +533,27 @@ Response body:
 		   "networks": {"vlan/native": "pxe", "vlan/235": "storage"}},
                   {"label": "nic2", 
 		   "macaddr": "12:34:56:78:90", 
-		   "networks":{"vlan/native": "public"}}],
+		   "networks":{}}],
+	 "metadata":{"EK":"pk"}
+	}
+
+
+Response body when run by an admin:
+
+There are two additional fields, `port` and `switch`.
```

Comment:
I'd meant for this to be up with the rest of the prose describing the response, maybe around here:

https://github.com/CCI-MOC/hil/pull/780/files#diff-0d56cc599e156c786c59f0d2bfafb9ddL523